participate.html

```diff
@@ -46,6 +46,13 @@ <h2>Participate!</h2>
                 know the answers.
               </li>
 
+	      <li>
+                Join our deal.II element chat 
+		<a href="https://matrix.to/#/#dealii:matrix.org">#dealii:matrix.org</a> at deal.II. 
+		There is a general chat for all	deal.II related questions and a lounge
+		for not so on-topic chats.
```

Comment:
```suggestion
                Join our deal.II <a href="https://matrix.to/#/#dealii:matrix.org">element chat</a>.
```
should be sufficient.

Reply:
updated accordingly.